interface/main/finder/dynamic_finder.php

```diff
@@ -33,11 +35,12 @@
 while ($row = sqlFetchArray($res)) {
     $colname = $row['option_id'];
     $title = xl_list_label($row['title']);
+    $title1 = ($title == xl('Full Name'))? xl('Name'): $title;
     $header .= "   <th>";
     $header .= text($title);
     $header .= "</th>\n";
-    $header0 .= "   <td align='center'><input type='text' size='10' ";
-    $header0 .= "value='' class='search_init' /></td>\n";
+    $header0 .= "   <td ><input type='text' size='20' ";
+    $header0 .= "value='' class='search_init' placeholder='".xla("Search by $title1")."'/></td>\n";
```

Comment:
This will be impossible for translators (ie. we can't created constants from this) and note the $title1 was already translated above. Maybe do something like:
```
$header0 .= "value='' class='search_init' placeholder='" . xla("Search by") . " "  . $title1 . "'/></td>\n";
```